config/jvm.options

```diff
@@ -79,3 +79,6 @@
 
 # Copy the logging context from parent threads to children
 -Dlog4j2.isThreadContextMapInheritable=true
+
+# Avoid Nashorn deprecation logs in JDK > 11
+-Dnashorn.args="--no-deprecation-warning"
```

Comment:
Testing this locally did not work with the quotes in place, but did without

Reply:
Fixed, I was testing with flag passed directly in my IDE